packages/material-ui/src/Tabs/Tabs.js

```diff
@@ -342,6 +342,7 @@ const Tabs = React.forwardRef(function Tabs(props, ref) {
     debounce(() => {
       updateScrollButtonState();
     }),
+    []
```

Comment:
I'm pretty sure this is an eslint bug. It should complain about `updateScrollButtonState` missing as a dependency. But that always changes during render. I think we can just remove the `useCallback`?

Reply:
Probably because it isn't how you're supposed to use `useCallback`. For factories you'd almost always want to use `useMemo` and then it would properly detect dependencies.

Reply:
What would be the correct solution? Should we replicate how we should the debounce issue from other components?

Reply:
I don't know which is why I recommended starting with `useMemo`.